examples/aws/cloudformation/README.md

```diff
@@ -1,4 +1,9 @@
-# AWS CloudFormation based provisioning example.
+
+## Important Notice
```

Comment:
Kind of awkward to have an `h2` before an `h1` no?

Reply:
Is `h1` preferred here?